rust/kernel/lib.rs

```diff
@@ -60,7 +60,7 @@ pub mod mm;
 #[cfg(CONFIG_NET)]
 pub mod net;
 pub mod pages;
-pub mod power;
+pub mod pm;
```

Comment:
Does this compile without `CONFIG_PM`? (from a quick look, all the functions called below are available even if just failing stubs, but just making sure you checked).

Reply:
Yeah, that is what I meant by the "failing stubs" (for the ones that return a value) that I saw in `pm.h`/`pm_runtime.h`, but it is easy sometimes to miss something that does not end up compiling, so I thought I would ask. Thanks for checking!